sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_single_node/language-modeling/gpt-2.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
```

Comment:
<p>I haven't been informed that the native PyTorch support became available. We added the new <code>pytorchxla</code> as distribution strategy only for the HugginigFace estimator. That's also how we documented in the <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/training-compiler-enable-pytorch.html" target="_blank">dev guide</a> and the <a href="https://sagemaker.readthedocs.io/en/stable/frameworks/huggingface/sagemaker.huggingface.html#hugging-face-estimator" target="_blank">SageMaker HuggingFace estimator's docstring</a>. We didn't added <code>pytorchxla</code> to the <a href="https://sagemaker.readthedocs.io/en/stable/frameworks/pytorch/sagemaker.pytorch.html" target="_blank">SageMaker PyTorch estimator</a>.</p><p> </p><p>Is this statement true?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3593/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yes, pytorchxla support is restricted to HuggingFace estimator with the Training Compiler enabled. On an unrelated note, pytorchddp support is currently only available for the PyTorch estimator.

Training compiler does not support PyTorch estimator.

Reply:
Then the text here should be fixed/reverted

Reply:
<p>The text already reflects the use of PyTorch estimator for native and HuggingFace estimator with training compiler. Can you make a suggestion ?</p>

Reply:
<p>I think the statement "We use a <code>PyTorch</code> estimator for native PyTorch" is wrong for any case.</p>

Reply:
<p>Even without the distribution strategy pytorchxla, we do not support the native PyTorch, as far as I understand.</p><p> </p>

Reply:
<p>The SageMaker PyTorch estimator class does not have the arg for SMTC. https://sagemaker.readthedocs.io/en/stable/frameworks/pytorch/sagemaker.pytorch.html</p>

Reply:
<p>Currently we support native TF through the SM TensorFlow estimator, HF-PT&amp;HF-TF through the SM HuggingFace estimator. But not the native PyTorch.</p>